index.html

```diff
@@ -0,0 +1,30 @@
+<!Doctype <!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Andrew Lyon</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" media="screen" href="main.css" />
+    <script src="main.js"></script>
+</head>
+<body>
+    <ul class="nav">
+        <li><a href="#">Home</a></li>

```

Comment:
I don't see any other pages, but this is where you would put the file path instead of the # , and you should only have 3 links.